xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -234,10 +234,15 @@ com.xwiki.macros.internal.recentlyupdated.RecentlyUpdatedThemeEnum.social=User,
 com.xwiki.macros.internal.recentlyupdated.RecentlyUpdatedThemeEnum.sidebar=Title, time
 
 ## View Files macro
-com.xwiki.macros.viewfile.DisplayType.full=Full
-com.xwiki.macros.viewfile.DisplayType.thumbnail=Thumbnail
+rendering.macro.view-file.description=Show a file using PDF Viewer Macro or Office Viewer.
+rendering.macro.view-file.parameter.attFilename.name=Att--filename
+rendering.macro.view-file.parameter.attFilename.description=Alias of File name (here for compatibility reasons).
+rendering.macro.view-file.parameter.name.name=File name
+rendering.macro.view-file.parameter.name.description=The attachment reference to display.
+rendering.macro.view-file.parameter.page.name=Page
+rendering.macro.view-file.parameter.page.description=The source page of the attachment.
```

Comment:
I would add more details to explain that this is needed for the file param and changing it will reset the file value